magprime/templates/regextra.html

```diff
@@ -13,6 +13,9 @@
             $("input:radio[name='amount_extra'][value=250]").parents('.btn').append($('#canvas_print'));
             $('#canvas_print').on('click', function(e) { e.stopPropagation(); })
         }
```

Comment:
FYI, ``$('.foo')`` is a jQuery object which is always truthy even if it contains zero elements.  So this if-statement is equivalent to just saying ``if (true)``.  Unlike in Python, where empty collections such as ``[]`` and ``{}`` are falsey, in Javascript they are truthy.

If you want to do an if-check on whether or not something exists, you can instead say ``if ($('.foo').size())``